### DISCUSSION TITLE
Is there anyway for Stories interaction (play function) to not autoplay?

### DISCUSSION THREAD

**Discussion by ramarivera** [storybookjs/storybook#23616] [Help]
I would like to run my interactions manually by clicking on some play button, instead of having the story play function execute automatically when opening the story. Is this intended behavior?

**Comment by dgalimidi** [comment]
I would like the same thing. @ramarivera, have you found a solution to this?

**Reply by ramarivera** [reply]
Unfortunately, not

**Comment by alexis-regnaud** [comment]
Same here, would be useful and less disturbing to be able to run it manually

**Comment by eeyzkoj** [comment]
good question, I am also curious about that, I posted a question here: https://github.com/storybookjs/storybook/discussions/24785, hopefully, to get a full answer on how the interaction testing intended to be :)

**Comment by abhatia1994** [comment]
There is a setting `autoplay`, its only for docs though. 
https://storybook.js.org/docs/api/doc-block-story#autoplay

**Comment by rikisamurai** [comment]
is there any solutions?

**Comment by manandkumaar** [comment]
We got a use case to enable or disable conditionally since play functions runs automatically, our Dev/QA/Consumers folks are facing difficulty to play around the components. Please share if there is any workaround to stop running play functions automatically.

Having the following flexibility would certainly help everyone - 

1. any option to run interaction test (play functions) conditionally only when interactions tab is select/active in addons panel 
2. Global config to stop running play functions
3. Story level to stop running play functions

**Comment by oscarviq** [comment]
There really should be a way to only run play on demand...

**Comment by kendallgassner** [comment]
Hey folks! I ran into a similar issue a month ago and remembered checking out this discussion, so I thought I’d drop back in and share the [workaround](https://github.blog/engineering/user-experience/how-to-make-storybook-interactions-respect-user-motion-preferences/) I came up with. 

Hope this helps someone!
